checks/redundant_items.lua

```diff
@@ -2,6 +2,8 @@
 
 --[[ Comparing the second part of the mane ]]
 
+local modutils = dofile(minetest.get_modpath("qa_block").."/modutils.lua")
```

Comment:
This file is also "dofile"-ed in `./no_sounds.lua`. Does this mean it actually gets ran twice?

Reply:
(more than twice - I found at least one more)

Reply:
Yeah. I've seen this lua pattern in other places too. I think it's a way to keep a module local instead of global (like if you include a module that includes another one). It might prevent errors in case the same mod is included multiple times.

Reply:
Lua has a built-in `require` feature that prevents the code from running multiple times while still respecting this design pattern. Would that work in the context of minetest? I don't see it used often.

It's not a super big deal really, but it does have both a startup and a memory cost to dofile multiple times. (in the case of memory, the used exports of each call are kept in memory till shutdown, meaning there would likely be a few duplicate functions in memory)

Reply:
I must admin, I'm not very well versed in Lua best-practices. The pattern used here makes sense from a scope perspective, but it may not be good for performance as you say, taking more memory for possibly having more objects loaded, and more dofile statements.

Reply:
I believe it would be best to run this script once and shove it into a new `qa_block.modutils` table value. It's internal (and undocumented) API, hence should not be touched by external mods.

Reply:
I agree, that would be best. However, I think checks scripts don't have access to the global qa_block. I thought that was why someone created the modutils; a small toolbox to be shared by the checks scripts.

If this is the case, then using require instead of dofile might solve the multiple inclusion issue. 

I won't be able to test any changes until Thursday as I don't have access to a proper computer at the moment

Reply:
It works! Pushed a commit with the changes.